src/libs/runOnLiveMarkdownRuntime/index.native.tsx

```diff
@@ -0,0 +1,8 @@
+import {getWorkletRuntime} from '@expensify/react-native-live-markdown';
+import {runOnRuntime} from 'react-native-reanimated';
+
+function runOnLiveMarkdownRuntime<Args extends unknown[], ReturnValue>(worklet: (...args: Args) => ReturnValue) {
```

Comment:
```suggestion
function runOnLiveMarkdownRuntime<Args extends unknown[], ReturnType>(worklet: (...args: Args) => ReturnType) {
```

Because we are operating in "type space" kinda here, then maybe this is more accurate. What do you think?